dmt/report/mixins.py

```diff
@@ -95,10 +95,19 @@ def calc_interval(self):
         self.interval_end = aware_end
 
     def get_params(self):
-        self.range_days = int(
-            self.request.GET.get('range', self.range_days))
-        self.offset_days = int(
-            self.request.GET.get('offset', self.offset_days))
+        try:
+            self.range_days = int(
+                self.request.GET.get('range', self.range_days))
+        except ValueError:
+            # Just use the default if we can't parse the urlparam.
+            self.range_days = 31
+
+        try:
+            self.offset_days = int(
+                self.request.GET.get('offset', self.offset_days))
```

Comment:
The sentry error seems to be happening because the default value self.offset_days is not initialized. Would it make more sense to simply init self.offset_days (and self.range_days)? Or, am I missing some complexity here?